docs/admin/install/helm-install/google-gke.md

```diff
@@ -3,6 +3,186 @@ tags:
+        ```
+
+      
+        
+        Add this to override.yaml:
+
+        ```yaml
+        global:
+            lbIp: #Add IP from previous command
+            isFqdnRegistered: false
+        ```
+
+    - FQDN/domain registered:
+    
+        If the FQDN for jans i.e `demoexample.jans.org` is registered and globally resolvable, you have to map it to the loadbalancer IP created in the previous step by Nginx-Ingress.
+        More details in this [guide](https://medium.com/@kungusamuel90/custom-domain-name-mapping-for-k8s-on-gcp-4dc263b2dabe).
+
+        Add this to override.yaml:
+
+        ```yaml
+        global:
+            lbIp: "" #Add LoadBalancer IP from previous command
+            isFqdnRegistered: true
+            fqdn: demoexample.jans.org #CHANGE-THIS to the FQDN used for Jans
+        nginx:
+          ingress:
+              enabled: true
+              path: /
+              hosts:
+              - demoexample.jans.org #CHANGE-THIS to the FQDN used for Jans
+              tls:
+              - secretName: tls-certificate
+                hosts:
+                - demoexample.jans.org #CHANGE-THIS to the FQDN used for Jans
+        ```
+
+
+
+
+
+
+    -  LDAP/Opendj for persitence storage
+
+
+          Add this to override.yaml:
+          ```yaml
+          global:
+            cnPersistenceType: ldap
+            storageClass:
+              provisioner: kubernetes.io/gce-pd
+            opendj:
+              enabled: true
+          ```
+
+          So now configuring both LDAP and no-FQDN will look something like that:
+
+          ```yaml
+           global:
+             cnPersistenceType: ldap
+             lbIp: #Add Load Balancer IP from previous command
+             isFqdnRegistered: false
+             storageClass:
+               provisioner: kubernetes.io/gce-pd
+             opendj:
+               enabled: true
+          ```
+
+
+
+
+
+
+
+    - MySQL for persistence storage
+      ```
+      helm repo add bitnami https://charts.bitnami.com/bitnami
+      helm install my-release --set auth.rootPassword=Test1234#,auth.database=jans bitnami/mysql -n jans
+      ```
+
+        Create a file named `mysql.yaml` and this to it:
+      
+        ```yaml
+        global:
+          cnPersistenceType: sql
+        config:
+          configmap:
+            cnSqlDbName: jans
+            cnSqlDbPort: 3306
+            cnSqlDbDialect: mysql
+            cnSqlDbHost: my-release-mysql.jans.svc
+            cnSqlDbUser: root
+            cnSqlDbTimezone: UTC
+            cnSqldbUserPassword: Test1234#
+        ```
+
+          Install the MySQL database:
+            ```
+            helm install janssen janssen/janssen -n jans -f mysql.yaml
+            ```
+
+3.  Install Jans
+
+
+
+      After finishing all the tweaks to the `override.yaml` file, we can use it to install jans.
+
+      ```
+      helm repo add janssen https://docs.jans.io
```

Comment:
The repository is changing `https://docs.jans.io/charts` so please change it to that